blog/tests.py

```diff
@@ -21,21 +27,41 @@ def register(self, credentials):
         return view(request)
 
     def create_blog_request(self, body):
+        """
+        Creates a Blog Post.
+        """
         credentials = {
             'username': 'test',
             'email': 'abc@example.com',
             'password': '123'
         }
 
-        response = self.register(credentials)
+        self.register(credentials)
 
         factory = APIRequestFactory()
         url = '/api/blog'
 
         request = factory.post(url, body)
         return request
 
+    def create_blog(self):
```

Comment:
nit: since this is a helper function inside the test class we should start the name with underscore, e.g. `def _create_blog(self):`

Reply:
done.